config/chime_science_run_gpu.yaml

```diff
@@ -491,11 +542,10 @@ gpu:
   gpu_2:
     kotekan_stage: hsaProcess
     gpu_id: 2
-    updatable_gain_psr: /pulsar_gain
-    updatable_gain_frb: /frb_gain
-    source_ra:  [53.51337,44.51337,52.1633,60.26337,46.76336,54.63837,49.01337,60.26337,55.76337,55.76337]
-    source_dec: [54.6248916,54.7248916,54.7598915,54.7248916,54.6248916,54.6448916,54.5748916,54.5248916,54.5048916,54.4748916]
-    psr_scaling: [48,48,48,48,48,48,48,48,48,48]
+    updatable_config:
```

Comment:
The convention until now would be sth like:
```
gpu_2:
    updatable_config_gain_psr: /updatable_config/pulsar_gain
    ...
```

Reply:
I don't think that's true.   Because isn't there a function that works on all `updatable_config` block references and matches them to call backs?  Let me check.

Reply:
This section does follow the currently documented convention: 
``` * In the config block of a stage that wants to get updates of a specific
 * updatable block, either a key "updatable_config" with the full path to the updatable
 * block as a value has to exist, or an object called "updatable_config" with
 * a list of all updatable config blocks.
 *
 * Example:
 * ```
 * my_stage:
 *     updatable_config: "/foo/bar"
 * ```
 * or
 * ```
 * my_stage:
 *     updatable_config:
 *         bar: "/foo/bar"
 *         fu: "/foo/fu"
 * ```

Reply:
Oh, you're right. Thanks!